pkg/rancher-desktop/main/networking/index.ts

```diff
@@ -19,8 +19,7 @@ import { windowMapping } from '@pkg/window';
 const console = Logging.networking;
 
 export default async function setupNetworking() {
-  const session = Electron.session.defaultSession;
-  const httpsOptions: https.AgentOptions = { ...https.globalAgent.options };
+  const httpsOptions = { ...https.globalAgent.options };
```

Comment:
nit; can we call this `proxyOptions` because it's a little confusing to pass it as `httpsOptions` to `httpAgent`.

Reply:
I opted for `agentOptions` because it gets passed to `http.Agent` / `https.Agent`. (That is, _not_ the proxy agent.)  But yes, calling it `httpsOptions` isn't quite correct, so that was changed.